src/features/donnees-personnelles/pages/donnees-personnelles/donnees-personnelles.page.html

```diff
@@ -0,0 +1,137 @@
+<div class="container m-5">
+  <h1 class="my-5 mx-5">Données personnelles</h1>
+  <h3 class="my-5 mx-5 text-primary">Traitement des données à caractère personnel</h3>
+  <div class="row">
+    <div class="col-7 mx-5">
+      <p>
+        La plateforme "Cartographie nationale" est à l'initiative de l'Agence Nationale de la Cohésion des Territoires (ANCT),
+        placé sous la tutelle des ministres chargés de l'aménagement du territoire, des collectivités territoriales et de la
+        politique de la ville.
+      </p>
+    </div>
+  </div>
+</div>
+<div class="container m-5">
+  <h3 class="my-5 mx-5 text-primary">Finalité</h3>
+  <div class="row">
+    <div class="col-7 mx-5">
+      <p>
+        La plateforme sert à faciliter l’identification des structures de médiation numérique ainsi que l’orientation des
+        usagers finaux.
+      </p>
+    </div>
+  </div>
+</div>
+<div class="container m-5">
+  <h3 class="my-5 mx-5 text-primary">Données à caractère personnel traitées</h3>
+  <div class="row">
+    <div class="col-7 mx-5">
+      <p>La plateforme peut traiter les données à caractère personnel suivantes :</p>
+      <ul>
+        <li>
+          <strong>Données de localisation:</strong>
+          Adresse ;
+        </li>
+        <li>
+          <strong>Données d'hébergeur:</strong>
+          Identifiant de connexion ; Nature des opérations ;
+        </li>
+        <li>
+          <strong>Cookies</strong>
+        </li>
+      </ul>
+      <div class="ps-4">
+        <h3 class="my-5 text-primary">Base juridique des traitements de données</h3>
+        <h4 class="my-4 text-primary">a) Les données de localisation</h4>
+        <p>
+          Ce traitement de données est nécessaire à l’exercice d’une mission d’intérêt public ou relevant de l’exercice de
+          l’autorité publique dont est investi le responsable de traitement au sens de l’article 6-e du règlement (UE) 2016/679
+          du Parlement européen et du Conseil du 27 avril 2016 relatif à la protection des personnes physiques à l’égard du
+          traitement des données à caractère personnel et à la libre circulation de ces données.
+        </p>
+        <h4 class="my-4 text-primary">b) Données d'hébergeur</h4>
+        <p>
+          Ce traitement de données est nécessaire au respect d’une obligation légale à laquelle le responsable du traitement est
+          soumis au sens de l’article 6-c du règlement (UE) 2016/679 du Parlement européen et du Conseil du 27 avril 2016
+          relatif à la protection des personnes physiques à l’égard du traitement des données à caractère personnel et à la
+          libre circulation de ces données.
+        </p>
+        <p>L’obligation légale est posée par :</p>
+        <ul>
+          <li>
+            <a href="https://www.legifrance.gouv.fr/loda/id/JORFTEXT000000801164/2020-10-26/" target="_blank">
```

Comment:
```suggestion
            <a href="https://www.legifrance.gouv.fr/loda/id/JORFTEXT000000801164/2020-10-26/" target="_blank" rel="noopener noreferrer">
```
https://stackoverflow.com/questions/40593632/use-window-open-but-block-use-of-window-opener